source/_patterns/03-templates/50-borrower-dashboard/51-borrower-dashboard.mustache

```diff
@@ -49,11 +49,14 @@
 					{{> 01-molecules/15-loan-sections/03-current-status-meter}}
 					{{#hasAutoRepayment}}
 						<div class="panel callout">
-							{{#nextRepayment}}
+							{{#nextScheduledAutoRepayment}}
```

Comment:
Nice swap of old nextRepayment variables with new nextScheduledAutoRepayment variables -- lets us transparently display the next scheduled values for auto repayments. 💯